control/autoware_control_toolbox/include/splines/bspline_interpolator_templated.hpp

```diff
@@ -0,0 +1,730 @@
+// Copyright 2018-2022 The Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
As Horibe-san said, we have interpolation package.
What do you think about moving this PR's spline implementation to interpolation package.

Reply:
I'm not familiar with your project, but I got it.
I'll leave how to deal with this PR to @TakaHoribe !